
Files already downloaded and verified
Traceback (most recent call last):
  File "/usr/project/xtmp/yz705/ICL-Experiments/./experiment.py", line 72, in <module>
    experiment_base(dataset, p_bursty)
  File "/usr/project/xtmp/yz705/ICL-Experiments/./experiment.py", line 61, in experiment_base
    trainer.train()
  File "/usr/project/xtmp/yz705/ICL-Experiments/train.py", line 232, in train
    icl_batch = _convert_dict(next(iter(self.icl_eval_loader)))
  File "/home/users/yz705/.local/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 439, in __iter__
    return self._get_iterator()
  File "/home/users/yz705/.local/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 384, in _get_iterator
    return _SingleProcessDataLoaderIter(self)
  File "/home/users/yz705/.local/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 670, in __init__
    self._dataset_fetcher = _DatasetKind.create_fetcher(
  File "/home/users/yz705/.local/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 79, in create_fetcher
    return _utils.fetch._IterableDatasetFetcher(dataset, auto_collation, collate_fn, drop_last)
  File "/home/users/yz705/.local/lib/python3.10/site-packages/torch/utils/data/_utils/fetch.py", line 21, in __init__
    self.dataset_iter = iter(dataset)
  File "/usr/project/xtmp/yz705/ICL-Experiments/train.py", line 23, in __iter__
    return iter(self.generator())
TypeError: Trainer.__init__.<locals>.<lambda>() missing 1 required positional argument: 'x'
Traceback (most recent call last):
  File "/usr/project/xtmp/yz705/ICL-Experiments/./experiment.py", line 72, in <module>
    experiment_base(dataset, p_bursty)
  File "/usr/project/xtmp/yz705/ICL-Experiments/./experiment.py", line 61, in experiment_base
    trainer.train()
  File "/usr/project/xtmp/yz705/ICL-Experiments/train.py", line 232, in train
    icl_batch = _convert_dict(next(iter(self.icl_eval_loader)))
  File "/home/users/yz705/.local/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 439, in __iter__
    return self._get_iterator()
  File "/home/users/yz705/.local/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 384, in _get_iterator
    return _SingleProcessDataLoaderIter(self)
  File "/home/users/yz705/.local/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 670, in __init__
    self._dataset_fetcher = _DatasetKind.create_fetcher(
  File "/home/users/yz705/.local/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 79, in create_fetcher
    return _utils.fetch._IterableDatasetFetcher(dataset, auto_collation, collate_fn, drop_last)
  File "/home/users/yz705/.local/lib/python3.10/site-packages/torch/utils/data/_utils/fetch.py", line 21, in __init__
    self.dataset_iter = iter(dataset)
  File "/usr/project/xtmp/yz705/ICL-Experiments/train.py", line 23, in __iter__
    return iter(self.generator())
TypeError: Trainer.__init__.<locals>.<lambda>() missing 1 required positional argument: 'x'